recipe/meta.yaml

```diff
@@ -9,26 +9,36 @@ package:
 source:
   fn: {{ filename }}
   url: https://github.com/rdkit/rdkit/archive/{{ filename }}
-  sha256: 38e6fb9f063b6132310f17e654f2c4350876f9164b0a17b49fe3df7d0555a744
+  sha256: edd30682cc3031cf3f2b1a400f453629db332a1018f355cd3f7ff76b2f7f5398
 
 build:
   number: 0
-  skip: true  # [aarch64]
+  # We skip s390x because it's not supported, and win because of boost weirdness.
+  # win SHOULD work though, and this recipe does build with boost from CF so it's
+  # something we'll need to fix at some point.
+  skip: True # [s390x or win]
+  skip: True # [py<37]
   missing_dso_whitelist:
     - '*/RDKit*dll'  # [win]
+  run_exports:
+    - {{ pin_subpackage('rdkit', max_pin='x.x.x') }}
+  ignore_run_exports:
+    # This is not required by any library, it's just exported by cairo.
+    - pixman
 
 requirements:
   build:
+    - {{ compiler('cxx') }}
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
```

Comment:
Cross-plartform dependencies can be removed

Reply:
Oops, I forgot about that. Fixed.